src/dropin.js

```diff
@@ -19,6 +19,86 @@ var svgHTML = fs.readFileSync(__dirname + '/html/svgs.html', 'utf8');
 var DEFAULT_CHECKOUTJS_LOG_LEVEL = 'warn';
 var VERSION = process.env.npm_package_version;
 
+/**
+ * @typedef {object} Dropin~cardPaymentMethodPayload
+ * @property {string} nonce The payment method nonce.
+ * @property {object} details Additional account details.
+ * @property {string} details.cardType Type of card, ex: Visa, MasterCard.
```

Comment:
I would consider using `e.g.` since that's what's used [elsewhere](https://github.com/braintree/braintree-web-drop-in/pull/167/files#diff-1fdf421c05c1140f6d71444ea2b27638R21)